tools/db_bench_tool.cc

```diff
@@ -5094,7 +5092,7 @@ class Benchmark {
             seek++;
             read++;
             if (single_iter->Valid() && single_iter->key().compare(key) == 0) {
-              seek_found++;
+              found++;
```

Comment:
why

Reply:
`seek_found` is never used. From line 5117, I think the intent of the original author was to use them interchangeably .